Resources/doc/index.rst

```diff
@@ -9,7 +9,7 @@ DoctrineMongoDBBundle
 
 The `MongoDB`_ Object Document Mapper (ODM) is much like the Doctrine2 ORM
 in its philosophy and how it works. In other words, like the `Doctrine2 ORM`_,
-with the Doctrine ODM, you deal only with plain PHP objects, which are then
+with the MongoDB ODM, you deal only with plain PHP objects, which are then
 persisted transparently to and from MongoDB.
```

Comment:
I would leave like this for consistency with the preceding part of the sentence, but you do have a point, and I think the issue could be fixed like this:

```suggestion
persisted transparently to and from MongoDB in our case.
```

Reply:
yes agreed, lets do this 👍